### PR TITLE
SAK-45873 Calendar: Import Events: Microsoft Outlook Comma-separated File Does Not Import Titles

### PR DESCRIPTION
**Pull request by jesusmmp**
https://sakaiproject.atlassian.net/browse/SAK-45873

### CODE REVIEW

**Comment by mpellicer** [comment]
Why keep the 'import.outlook.charset' in the translation files if all of them have the same value? Do you think we can do a cleanup here?

File: calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/readers/CSVReader.java [hunk 1]
```diff
@@ -176,6 +177,7 @@ protected String[] parseLineFromDelimitedFile(String line)
 				DELIMITER_TOKEN_IN_REGEX,
 				getHexStringForDelimiter());
 
+		line = CSVReader.removeUTF8BOM(line);
```
Comment (ern) [inline calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/readers/CSVReader.java:180]: what about if someone uploaded a file with some other Byte Order Mark, probably a good idea to handle other BOM types as well?
see https://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/input/BOMInputStream.html

Comment (jesusmmp) [reply calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/readers/CSVReader.java]: Done!

**Comment by jesusmmp** [comment]
> Why keep the 'import.outlook.charset' in the translation files if all of them have the same value? Do you think we can do a cleanup here?

Done!

File: calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/readers/OutlookReader.java [hunk 2]
```diff
@@ -156,14 +157,12 @@ public String importStreamFromDelimitedFile(
...
-		InputStreamReader inStreamReader = null;
-		try {
-			inStreamReader = new InputStreamReader(stream, rb.getString("import.outlook.charset"));
-			
-		} catch (UnsupportedEncodingException e) {
-			inStreamReader = new InputStreamReader(stream);
-		}
-		BufferedReader bufferedReader = new BufferedReader(inStreamReader);
+		//Detect and exclude a UTF-8 BOM
+		BOMInputStream bomIn = new BOMInputStream(stream);
```
Comment (jesusmmp) [inline calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/readers/OutlookReader.java:161]: I don't know if this one exclude all BOM... any idea?

**Review by ern — APPROVED** [review]
Nice!!!

**Pull request merged**